File: example/src/main/java/ru/virgil/example/user/UserDetailsDto.java
package ru.virgil.example.user;

import lombok.Data;
import lombok.EqualsAndHashCode;
import ru.virgil.example.system.IdentifiedDto;

@EqualsAndHashCode(callSuper = true)
@Data
public class UserDetailsDto extends IdentifiedDto {

    private String name;

}

File: example/src/main/java/ru/virgil/example/user/UserDetailsDto.kt
package ru.virgil.example.user

import ru.virgil.example.system.dto.IdentifiedDto
import java.time.LocalDateTime
import java.util.*

class UserDetailsDto(
    override var createdAt: LocalDateTime?,
    override var updatedAt: LocalDateTime?,
    override var uuid: UUID?,
    var name: String? = null,
) : IdentifiedDto
